traefik.go

```diff
@@ -23,6 +23,7 @@ import (
 	"github.com/mailgun/oxy/cbreaker"
 	"github.com/mailgun/oxy/forward"
 	"github.com/mailgun/oxy/roundrobin"
+	// "github.com/spf13/viper"
```

Comment:
damned, you're too quick @ldez :stuck_out_tongue_closed_eyes: 
I was looking for gifs.. :sweat_smile: